website/content/en/status/report-2023-04-2023-06/capsicum-ktracing.adoc

```diff
@@ -0,0 +1,230 @@
+  1926 unzip    NAMI  "/usr/local/etc/libmap.d"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/var/run/ld-elf.so.hints"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libarchive.so.7"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/usr/lib/libarchive.so.7"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libc.so.7"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libz.so.6"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libbz2.so.4"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/usr/lib/libbz2.so.4"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/liblzma.so.5"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/usr/lib/liblzma.so.5"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libbsdxml.so.4"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libprivatezstd.so.5"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/usr/lib/libprivatezstd.so.5"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libcrypto.so.111"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libmd.so.6"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/lib/libthr.so.3"
+  1926 unzip    CAP   system call not allowed: readlink
+  1926 unzip    NAMI  "/etc/malloc.conf"
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/dev/pvclock"
+  1926 unzip    NAMI  "foo.zip"
+  1926 unzip    CAP   openat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    CAP   system call not allowed: open
+  1926 unzip    NAMI  "/etc/localtime"
+  1926 unzip    NAMI  "bar"
+  1926 unzip    CAP   fstatat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    CAP   system call not allowed: mkdir
+  1926 unzip    NAMI  "bar"
+  1926 unzip    NAMI  "bar"
+  1926 unzip    CAP   fstatat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    NAMI  "bar/bar.txt"
+  1926 unzip    CAP   fstatat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    NAMI  "bar/bar.txt"
+  1926 unzip    CAP   openat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    NAMI  "baz"
+  1926 unzip    CAP   fstatat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    CAP   system call not allowed: mkdir
+  1926 unzip    NAMI  "baz"
+  1926 unzip    NAMI  "baz"
+  1926 unzip    CAP   fstatat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    NAMI  "baz/baz.txt"
+  1926 unzip    CAP   fstatat: restricted VFS lookup: AT_FDCWD
+  1926 unzip    NAMI  "baz/baz.txt"
+  1926 unzip    CAP   openat: restricted VFS lookup: AT_FDCWD
+----
+
+The violation tracing output for `unzip` is more akin to what a developer would see when tracing their own program for the first time.
```

Comment:
As above, maybe use `man:unzip[1]` in place of each occurrence of `` `unzip` ``.